views/index.jade

```diff
@@ -11,7 +11,7 @@ block content
         h1 SPF.IO Songbase
         p Find Public Domain and CC0 Worship Song in Multiple Language
 
-        form.search-form(method="post" action="#")
+        form.search-form(method="post" action='')
```

Comment:
making it a blank action is okay, but you can also do an e.preventDefault() on the submission button/event so that it doesn't submit the form at all.